_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -436,7 +436,7 @@ ggplot() +
 > > are magenta, and we will set line width so that the first
 > > factor level is thicker than the others.
 > >
-> > ```{r harv-paths-bike-horses}
+> > ```{r harv-paths-bike-horses fig.cap="Roads and trails in the area highlighting paths where horses and bikes are allowed."}
```

Comment:
```suggestion
> > ```{r harv-paths-bike-horses, fig.cap="Roads and trails in the area highlighting paths where horses and bikes are allowed."}
```